Frameworks/UIKit/UINibUnarchiver.mm

```diff
@@ -203,7 +203,13 @@ static id constructObject(UINibUnarchiver* self, Object* pObj) {
         id classId = pObj->classType; // objc_getClass(pObj->className);
         assert(classId != nil);
 
-        pObj->cachedId = [classId alloc];
+        // TODO: #1105 alloc without init is invalid, and not all classes support alloc-init anyway
```

Comment:
Seems... undesirable?
[https://developer.apple.com/reference/foundation/1587930-nsallocateobject](https://developer.apple.com/reference/foundation/1587930-nsallocateobject)
"This function is deprecated and unavailable for use with ARC."

---
In reply to: [82673565](https://github.com/Microsoft/WinObjC/pull/1106#discussion_r82673565) [](ancestors = 82673565)